### DISCUSSION TITLE
ScriptReference/Bounds-size

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#43] [Notes]
# ScriptReference/Bounds-size

https://docs.unity3d.com/ScriptReference/Bounds-size.html

**Comment by vertxxyz** [comment]
Calls to [`extents`](https://docs.unity3d.com/ScriptReference/Bounds-extents.html) will be cheaper than `size` as size is treated as a multiple of extents.
Setting it will call `extents = value * 0.5f,` and getting it will return `extents * 2f`.
It's generally good practice to cache calls to properties if you're constructing a component vector because of situations like these.
Optimisations like these are only relevant in hot code paths.
